client-core/src/service/sync_state_service.rs

```diff
@@ -14,20 +11,23 @@ pub struct SyncState {
     pub last_block_height: u64,
     /// last app hash
     pub last_app_hash: String,
-    /// current trusted state for lite client verification
-    pub trusted_state: lite::TrustedState,
+    /// last block hash
+    pub last_block_hash: String,
```

Comment:
`String` is more compatible with the light client APIs?
`SyncState` isn't directly returned from RPC, so these could have more precise types / no need to store the hex string?

Reply:
No, it's just `SyncState` already using `String` to represent hashes, so I kept that tradition...